<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="example" uuid="0ddddf5d-1525-4d57-8dc9-f2f0e7ac2f9b">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:D:\Users\Administrator\AppData\Local\Programs\PycharmWorkSpace\Test_Dev\WindDemos\example.db</jdbc-url>
      <working-dir>$ProjectFileDir$</working-dir>
    </data-source>
    <data-source source="LOCAL" name="check_valuation_table" uuid="ffc6cdf8-a2d9-4074-936c-d04cea9a6a9a">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:D:\Users\Administrator\AppData\Local\Programs\PycharmWorkSpace\Test_Dev\testException\check_valuation_table.db</jdbc-url>
      <working-dir>$ProjectFileDir$</working-dir>
    </data-source>
  </component>
</project>